web-server/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "web-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:dev": "nodemon src/app.js -e js,hbs"
+  },
+  "keywords": [],
+  "author": "",
```

What do you think if we put here your name and email following the next structure
```suggestion
  "author": "Linda Zapata <lvzh90@gmail.com>",
```